scipy/fft/_realtransforms.py

```diff
@@ -621,6 +621,19 @@ def dst(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False, workers=None,
     The (unnormalized) DST-IV is its own inverse, up to a factor :math:`2N`. The
     orthonormalized DST-IV is exactly its own inverse.
 
+    Examples
+    --------
+    Compute the DST of a simple 1D array:
+
+    >>> import numpy as np
+    >>> from scipy.fft import dst
+    >>> x = np.array([1, 2, 3, 4])
+    >>> dst(x, type=2)
+    array([13.06562965, -5.65685425,  5.411961  , -4.        ])
```

Comment:
What do you think about using an input `x`, which could by verified by hand, like:
```suggestion
    >>> x = np.array([1, -1, 1, -1])
    >>> dst(x, type=2)
    array([0., 0., 0., 8.])
```